src/main/java/org/jboss/tools/intellij/exhort/TelemetryService.java

```diff
@@ -21,4 +25,16 @@ public class TelemetryService {
     public static TelemetryMessageBuilder instance() {
         return INSTANCE.builder.get();
     }
+
+    public static void sendTelemetryEvent(PsiFile file, String recommendedVersion, DependencyReport report, String actionName) {
```

Comment:
```suggestion
    public static void sendPackageUpdateEvent(PsiFile file, String packageName, String packageVersion, String actionName) {
```
I think the method name is too generic. This method only sends package and version that are changed.
Maybe, rename it to `sendPackageUpdateEvent`.
The third argument can be changed to `String`.

Reply:
done